packages/nx-plugin/src/generators/app/versions.ts

```diff
@@ -1,5 +1,9 @@
+export const MINIMUM_SUPPORTED_ANGULAR_VERSION = '15.0.0';
 // V16
 // dependencies
+export const V16_ANGULAR = '16.0.0';
+export const V16_NX_DEVKIT = '16.0.3';
```

Comment:
Pin all Nx deps to ^16.0.0 so it will pick up the latest release

```suggestion
export const V16_NX_DEVKIT = '^16.0.0';
```

Reply:
Should be addressed now